### PR TITLE
Dev: Remove the ignore errors of `Encryption` and `Entity` in phpstan.

### PR DESCRIPTION
**Pull request by ya861220**
**Description**
See #7731 

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Entity/Entity.php [hunk 1]
```diff
@@ -45,9 +45,16 @@ class Entity implements JsonSerializable
...
      *  ];
+     *
+     * @var array<string, string>
      */
     protected $datamap = [];
 
+    /**
+     * The date fields.
+     *
+     * @var array<string>
```
Comment (datamweb) [inline system/Entity/Entity.php:56]: What do you think about using `list<string>`?

Comment (paulbalandan) [inline system/Entity/Entity.php:56]:
```suggestion
     * @var array<int, string>
     * @phpstan-var list<string>
```

Comment (ya861220) [reply system/Entity/Entity.php]: Ok. Ths!

**Review by paulbalandan — APPROVED** [review]

**Comment by kenjis** [comment]
By the way, what do you mean by `dev:` in the commit messages? > @ya861220 @ping-yee @kai890707 

See https://www.conventionalcommits.org/en/v1.0.0/
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

**Comment by ping-yee** [comment]
It's our misunderstanding and we will follow the semantics commit message in later commits.
Appreciate it! @kenjis

**Review by kenjis — APPROVED** [review]

**Pull request merged**